packages/lab/stories/overlay/overlay.stories.tsx

```diff
@@ -1,83 +1,242 @@
-import { Button } from "@salt-ds/core";
+import {
+  Button,
+  Checkbox,
+  CheckboxGroup,
+  StackLayout,
+  Tooltip,
+} from "@salt-ds/core";
+import { Meta } from "@storybook/react";
+import React, { ChangeEvent } from "react";
+
 import {
   Overlay,
+  OverlayPanel,
   OverlayProps,
-  useOverlay,
-  UseOverlayProps,
+  OverlayTrigger,
 } from "@salt-ds/lab";
-import { Meta, StoryFn } from "@storybook/react";
 
 import "./overlay.stories.css";
+import { useOverlayContext } from "../../src/overlay/OverlayContext";
 
 export default {
   title: "Lab/Overlay",
   component: Overlay,
 } as Meta<typeof Overlay>;
 
-const OverlayTemplate: StoryFn<OverlayProps & UseOverlayProps> = (props) => {
-  const { placement, open, ...rest } = props;
-  const { getTriggerProps, getOverlayProps } = useOverlay({ placement, open });
-
+const OverlayContent = () => {
+  const { id } = useOverlayContext();
```

Comment:
Why would we expose this instead of allowing the user to generate an id and label everything correctly?

Reply:
👍🏻 updated to use user generated ID